app/templates/verify.hbs

```diff
@@ -1,9 +1,9 @@
 {{#if this.success}}
   <div class="ui {{if this.isLoading 'loading'}} icon info message eight wide column center aligned">
-    <p>Your email has been verified successfully!{{#if (not this.session.isAuthenticated)}} Please login to continue. {{/if}}</p>
+    <p>your email has been verified successfully! You now have full access to all feature.{{#if (not this.session.isAuthenticated)}} Please login to continue. {{/if}}</p>
```

Comment:
English sentences start with a capital letter

Reply:
Please use: 
Your account is unverified. Please verify it by clicking on the confirmation link that has been emailed to you.